src/colp/components/usetype_version_comparison_report.py

```diff
@@ -0,0 +1,76 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+from st_aggrid import AgGrid
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import pdb
+
+
+class UsetypeVersionComparisonReport:
+    def __init__(self, data):
+        self.usetype_changes = data["usetype_changes"]
+
+    def __call__(self):
+        st.subheader("Usetype Version Comparison")
+
+        version = st.sidebar.selectbox(
+            "Select a Version for Comparison", ["2022/06/16"]
+        )
+
+        self.current_usetype_changes = self.filter_by_version(
+            self.usetype_changes, version
+        )
+
+        st.markdown(
+            f"""
+            This report compares the usetype distribution between the previous and selected versions.
+            Because there are a high number of usetypes, we have included a slider to be able to more clearly view a subsection.
+
+            **Current Version:** {version}
+
+            **Previous Version:** {self.previous_version(self.current_usetype_changes)}
```

Comment:
Current and previous version are listed in different formats

Reply:
IMO this is a problem that should be corrected in the underlying dataset, which has them in different formats